patterns/workers/errors.md

```diff
@@ -0,0 +1,56 @@
+= Tratamento de Erros
+
+== rescue_handler
+
+Para mostrar o erro de um worker no Sentry e nos logs, mas não permitir retry manual no Sidekiq, use `rescue_handler`. Ex.:
```

Comment:
talvez precisa-se ficar claro q isso vai fazer o sidekiq considerar todas as execuções do worker como sucesso, independendo do q ocorreu durante o processamento, que torna a métricas do sidekiq imprecisas

A opção para somente skippar a dead queue é  [dead: false](https://github.com/mperham/sidekiq/wiki/Error-Handling#:~:text=You%20can%20disable%20a%20job%20going%20to%20the%20Dead%20set)